MAUI/DataGrid/sorting.md

```diff
@@ -0,0 +1,252 @@
+
+    <syncfusion:SfDataGrid.SortColumnDescriptions>
+        <syncfusion:SortColumnDescription ColumnName="OrderID" SortDirection="Ascending" />
+    </syncfusion:SfDataGrid.SortColumnDescriptions>
+</syncfusion:SfDataGrid>
+{% endhighlight %}
+
+{% highlight c# %}
+SortColumnDescription sortColumnDescription = new SortColumnDescription() { ColumnName = "OrderID", SortDirection = ListSortDirection.Ascending };
+this.sfDataGrid.SortColumnDescriptions.Add(sortColumnDescription);
+{% endhighlight %}
+{% endtabs %}
+
+![Programmatical sorting .NET MAUI DataGrid](Images\sorting\maui-datagrid-programmatic-sorting.png)
+
+## Different modes of sorting
+
+`SfDataGrid` sorts the data against one or more columns based on the `SfDataGrid.SortingMode` property. The `SortingMode` contains the following properties,
+
+* **Single** - It allows to sort only one column at a time.
+* **Multiple** - It allows to sort more than one column at a time.
+* **None** - None of the columns can be sorted.
+
+To apply sorting for multiple columns, tap the desired column headers after setting the `SfDataGrid.SortingMode` property to [Multiple](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.DataGridSelectionMode.html#Syncfusion_Maui_DataGrid_DataGridSelectionMode_Multiple).
+
+{% tabs %}
+{% highlight xaml %}
+<syncfusion:SfDataGrid x:Name="sfDataGrid" SortingMode="Multiple" ItemsSource="{Binding OrderInfoCollection}" >
+
+    <syncfusion:SfDataGrid.SortColumnDescriptions>
+        <syncfusion:SortColumnDescription ColumnName="OrderID" SortDirection="Ascending" />
+    </syncfusion:SfDataGrid.SortColumnDescriptions>
+</syncfusion:SfDataGrid>
+{% endhighlight %}
+{% endtabs %}
+
+![Multi sorting .NET MAUI DataGrid](Images\sorting\maui-datagrid-multi-sorting.gif)
+
+## Tri-state sorting
+
+In addition to sort the data in ascending/descending order, the `SfDataGrid` unsort the data in the original order by clicking the header again after sorting to descending order by setting the [SfDataGrid.AllowTriStateSorting](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.SfDataGrid.html#Syncfusion_Maui_DataGrid_SfDataGrid_AllowTriStateSorting) property to true. When this property is set, sorting in each column iterates through three sort states: ascending, descending, and unsort.
+
+{% tabs %}
+{% highlight c# %}
+this.sfDataGrid.AllowTriStateSorting = true;
+{% endhighlight %}
+{% endtabs %}
+
+![Tri-state sorting .NET MAUI DataGrid](Images\sorting\maui-datagrid-tristate-sorting.gif)
+
+## Show sort number
+
+The `SfDataGrid` provides support the sequence numbers to display the sorted columns during multi-column sorting by setting [SfDataGrid.ShowSortNumbers](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.SfDataGrid.html#Syncfusion_Maui_DataGrid_SfDataGrid_ShowSortNumbers) is set to true. This is applicable when the `SfDataGrid.SortingMode` property is `Multiple`.

```

Comment:
The `SfDataGrid` provides support to display the sequence numbers to denote the order of the column in which they are sorted during multiple column sorting by setting